docs/develop/Applications.md

```diff
@@ -0,0 +1,66 @@
+# Applications
+
+There are many different paths for creating applications that work using ElectionGuard. For many of these paths, there are reference implementations that can be forked or used as examples or used as packages. Many reference implementations include packaging or containerization for easy consumption. Below are some examples of paths a developer could take when developing using ElectionGuard. This is by no means the limit and some of these paths can be combined for different applications and use cases. 
+
+If you develop an app for ElectionGuard, we suggest you [utilize our badges][badges] to indicate which version of the specification your application targets. 
+
+## Paths
+
+### ElectionGuard Core
+
+An ElectionGuard core is an application that implements the base level [features][features] such as Ballot Encryption by implementing them according to the specification. The internal examples of this are the python and c++ reference implementations. There is a community example of this with the java port don by the community. 
+
+### Ballot Marking
+
+Ballot marking means marking ballots with a voters selections, for the uninitiated, a voting application. The app would display the contests in the manifest according to voter's ballot style and language and allow the voter to vote. The result would be a plaintext ballot.
+
+### Ballot Encryption
+
+Encrypting ballots is the base of ElectionGuard. This is a tool that takes a plaintext ballot and encrypts the ballot into an encrypted aka ciphertext ballot. 
+
+Two quick ways to start are by using the ElectionGuard [encryption nuget package][encryption nuget package] or or [python package][python package].
+
+### Ballot Box
+
+A ballot box simply takes the encrypted ballot of a voter and allows a user to submit the ballot as cast or spoil/challenge the ballot. This takes a ciphertext ballot and turns it into a submitted ballot that is aggregated into a tally.
```

Comment:
It's a noun. These are app descriptions. 

A ballot box app 
A ballot encryption app
A ballot marking app

Reply:
it is definitely a noun in this use case. 

A ballot box stores both the encrypted ballot of a voter and whether it should be treated as cast or challenged by the tally ceremony.

Reply:
Yea, not verb, that was incorrect.  I don't know English well enough to explain exactly why it sounds akward.  It's like saying a runner runs makes sense, but saying "a box takes ballots" is odd.  I guess a "box accept ballots"?  Maybe that's all that's necessary, but it still feels off to me.  Maybe because it's using the passive voice.

Reply:
RC's definition is really moving in the right direction.  To make it contain the word "is", how about:

> A ballot box is a single voter's encrypted ballot combined with meta-data about the vote such as whether it should be treated as cast or challenged by the tally ceremony

Reply:
I would agree to get rid of the word simply.